src/meta_schedule/database/database.cc

```diff
@@ -154,6 +154,59 @@ TuningRecord TuningRecord::FromJSON(const ObjectRef& json_obj, const Workload& w
   return TuningRecord(trace, workload, run_secs, target, args_info);
 }
 
+/******** Database ********/
+
+Optional<TuningRecord> DatabaseNode::QueryTuningRecord(IRModule mod, Target target) {
```

Comment:
We may want to add a target comparison function to distinguish between the target given here in the future.

Reply:
that makes sense!